How to generate Google API keys.md

```diff
@@ -25,14 +25,14 @@ Here's how to get its 3 access keys: `clientId`, `clientSecret`, `refreshToken`
 
 	<img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">
 
-0. Select **Other** (or **Desktop app** if available), enter `chrome-webstore-upload` and click **Create** 
+0. Select **Chrome App** (or **Desktop app** if available), enter `chrome-webstore-upload` and click **Create** 
```

Comment:
Is Other gone? I'll have to verify these steps because Chrome App existed already, but I didn't use it before.

```suggestion
0. Select **Chrome App** (**Desktop app** and **Other** also work, if Chrome App is missing), enter `chrome-webstore-upload` and click **Create** 
```

Reply:
Yeah last time I checked it's gone.